core/Pos/Crypto/SecretSharing.hs

```diff
@@ -120,77 +125,143 @@ instance Bi SecretProof =>
 ----------------------------------------------------------------------------
 
 -- | Extract ByteString from DhSecret.
-getDhSecret :: DhSecret -> ByteString
-getDhSecret (DhSecret s) = s
-
--- | Returns id of 'Share'.
-shareId :: Share -> ShareId
-shareId = Pvss.decryptedShareID . getShare
-
--- | Returns id of encrypted share.
-encShareId :: EncShare -> ShareId
-encShareId = Pvss.shareID . getEncShare
+getDhSecret :: Scrape.DhSecret -> ByteString
+getDhSecret (Scrape.DhSecret s) = s
 
 -- | Transform a Secret into a usable random value.
-secretToDhSecret :: Secret -> DhSecret
-secretToDhSecret = Pvss.secretToDhSecret . getSecret
+secretToDhSecret :: Secret -> Scrape.DhSecret
+secretToDhSecret = Scrape.secretToDhSecret . getSecret
 
 -- | Decrypt share using secret key. Doesn't verify if an encrypted
 -- share is valid, for this you need to use verifyEncShare.
```

Comment:
There is no `verifyEncShare` function anymore, so I guess this comment should be updated.

Reply:
Updated.